parts/language/lang_en.lua

```diff
@@ -858,19 +870,54 @@ return{
         {C.C,"Also try Orzmic!"},
         {C.C,"Also try osu!"},
         {C.C,"Also try Phigros!"},
+        {C.C,"Also try Puyo Puyo!"},
         {C.C,"Also try Rubik's cube!"},
         {C.C,"Also try Terraria!"},
+        {C.C,"Also try Touhou Project!"},
         {C.C,"Also try VVVVVV!"},
+        {C.G,"GREEN"},
+        {C.G,"Mac shortcut: Cmd-[=Go to the previous folder"},
+        {C.G,"Mac shortcut: Cmd-A=Select All items"},
+        {C.G,"Mac shortcut: Cmd-F=Find items"},
+        {C.G,"Mac shortcut: Cmd-Option-Delete=Delete the file immediately"},
+        {C.G,"Mac shortcut: Cmd-Q=Quit an app"},
+        {C.G,"Mac shortcut: Control-↑=Show mission control"},
+        {C.G,"Mac shortcut: Cmd-Tab=Switch among your open apps"},
+        {C.G,"Mac shortcut: Cmd-W=Close the front window"},
+        {C.G,"Mac shortcut: Cmd-Z=Undo the previous command"},
+        {C.G,"Mac shortcut: Cmd-,=Open preferences for the front app"},
+        {C.G,"Windows shortcut: Alt+F4=Close the active window"},
+        {C.G,"Windows shortcut: Alt+Tab=Switch among windows"},
+        {C.G,"Windows shortcut: Backspace=Go to the previous folder"},
+        {C.G,"Windows shortcut: Ctrl+Mouse Wheel=Zoom"},
+        {C.G,"Windows shortcut: Ctrl+A=Select all items"},
+        {C.G,"Windows shortcut: Ctrl+F=Find items"},
+        {C.G,"Windows shortcut: Ctrl+Tab=Switch among tabs"},
+        {C.G,"Windows shortcut: Ctrl+W=Close the active tab"},
+        {C.G,"Windows shortcut: shift+del=Permanently delete a file"},
```

Comment:
All shortcut tips: I was against them because they are highly software-dependent and totally unrelated to Techmino or :thinkZ:.